ethereum/smart-contract/contracts/interfaces/IGatewayNetwork.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.19;
+
+abstract contract  IGatewayNetwork {
+    // Ranges from 0% - 100%
+    /**
+     * @dev Struct that describes the fees of each 
+     */
+    struct NetworkFeesPercentage {
+        // Token address used to pay fees. Zero address for this mean native eth
+        address tokenAddress;
+        uint16 issueFee;
+        uint16 refreshFee;
+        uint16 expireFee;
+    }
+
+    /**
+     * @dev This struct represents data associated with the state of a gatekeeper network
+     */
+    struct GatekeeperNetworkData {
+        // the authority for this network
+        address primaryAuthority;
+
+        // Unique identifier for a network.
+        bytes32 name;
+        
+        // The default expiration time of passes on this network in seconds
+        uint passExpireTimeInSeconds;
```

Comment:
Does uint `0` have special meaning. We should comment  that here (I assume invite valid passes).

Reply:
The current implementation requires it to be greater than 0. Do we want to allow VC's to have an infinite lifetime?

Networks can set a timestamp far in the future to achieve that.

Reply:
Yep `0` means there is no expiration. I updated the comment to mention that